### DISCUSSION TITLE
Assets are not showing if user is not logged

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#4644] [Error / Bug Report] [answered]
Hi!
I found a bug and i love Wikijs and that why i think you would work to fix this bug.
Code as create issue:
```
if (!WIKI.auth.checkAccess(req.user, ["read:assets"], pageArgs)) {
      return res.sendStatus(403);
} 

```
File: wiki/server/controllers/common.js
line 633.

I already gave "read assets" to all guests but still images are not showing. 
When i comment the code, Images are back and showing.

Thanks!

**Comment by gdevenyi** [comment, marked as answer]
See https://github.com/Requarks/wiki/discussions/4589
